cloudferrylib/os/compute/nova_compute.py

```diff
@@ -710,10 +711,23 @@ def create_instance(self, nclient, **kwargs):
                   pprint.pformat(boot_args))
         created_instance = nclient.servers.create(**boot_args)
 
+        current_auth_user_id = created_instance.user_id
+        instance_user_id = kwargs['user_id']
+        flavor_obj = nclient.flavors.get(boot_args['flavor'])
+        flavor = {
+            'vcpus': flavor_obj.vcpus,
+            'ram': flavor_obj.ram,
+        }
+        tenant_id = created_instance.tenant_id
```

Comment:
Please move these changes inside `if self.config.migrate.keep_usage_quotas_inst:` statement, they are not used outside of it.